src/components/application_manager/rpc_plugins/app_service_rpc_plugin/include/app_service_rpc_plugin/app_service_rpc_plugin.h

```diff
@@ -47,7 +47,8 @@ class AppServiceRpcPlugin : public plugins::RPCPlugin {
   bool Init(app_mngr::ApplicationManager& application_manager,
             app_mngr::rpc_service::RPCService& rpc_service,
             app_mngr::HMICapabilities& hmi_capabilities,
-            policy::PolicyHandlerInterface& policy_handler) OVERRIDE;
+            policy::PolicyHandlerInterface& policy_handler,
+            resumption::LastState& last_state) OVERRIDE;
```

Comment:
@ShobhitAd Actually, we can't move these changes to separate PR because we need the `LastState` entity in rc_rpc_plugin for initialization `RCConsentManager`, which uses it for saving user consents into the filesystem.

In SDL [software architecture document](https://smartdevicelink.com/en/docs/sdl-core/master/software-architecture-document/components-view/) you can see in [ComponentView diagramm](https://d83tozu1c8tt6.cloudfront.net/media/builds/master/14/c3714fcd74d55ee3a7b38e61e4ff3952cf07194a/software-architecture-document/components-view/assets/ComponentView.png) that all plugins inheritances `RPCPlugin interface`. And if we need to initialize `RCRPCPlugin` with the `LastState` entity, we need to add the LastState interface to initialization all plugins.
In the current implementation, all plugins except `RCRPCPlugin` don't use this interface and mark it as `UNUSED`.
RCRPCPlugin uses the interface for initialization `RCConsentManager`, which has been added in scope of this feature